config/initializers/devise.rb

```diff
@@ -273,6 +273,30 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  dfe_sign_in_issuer_uri    = URI(ENV.fetch("DFE_SIGN_IN_ISSUER", "example"))
+  dfe_sign_in_identifier    = ENV.fetch("DFE_SIGN_IN_IDENTIFIER", "example")
+  dfe_sign_in_secret        = ENV.fetch("DFE_SIGN_IN_SECRET", "example")
+  dfe_sign_in_redirect_uri  = ENV.fetch("DFE_SIGN_IN_REDIRECT_URL", "example")
```

Comment:
Is there a reason we don't `URI` this, but we do on line 276?

Reply:
(uri on left, URL on right)

Reply:
Only the first one has to be a URI because it has to respond to `port` method. The latter does not need to